apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -1,13 +1,41 @@
-import { Keys } from './consts';
+import { Keys, ROOT_VIEW } from './consts';
 import { TESTPAGE_BUTTONS_SCROLLVIEWER } from '../../TestComponents/Common/consts';
 import { Attribute } from './consts';
 
 const DUMMY_CHAR = '';
+// The E2ETEST_PLATFORM environment variable should be set in the beforeSession hook in the wdio.conf file for the respective platform
+const PLATFORM = process.env['E2ETEST_PLATFORM'];
+if (PLATFORM) {
+  console.log(`Using platform: ${PLATFORM}`);
+}
 export const COMPONENT_SCROLL_COORDINATES = { x: -0, y: -100 }; // These are the offsets. Y is negative because we want the touch to move up (and thus it scrolls down)
 
+let _rootView: WebdriverIO.Element | null = null;
```

Comment:
I've never seen `null =  null` as a type declaration. Did you just mean `null`?

Reply:
Oh wait, you're defining the type and then immediately assigning it to `null`?

Reply:
Yeah, just a type definition saying this is either an Element or null (on startup).